openshift/ci-operator/knative-images/apiserver_receive_adapter/Dockerfile

```diff
@@ -2,5 +2,6 @@
 FROM openshift/origin-base
 USER 65532
 
+COPY ./cmd/apiserver_receive_adapter/kodata /var/run/ko
```

Comment:
I guess.... this does not cause any harm....  so I am OK.... but it worked before, without

Reply:
yeah, I could've deleted the kodata dir of it but let's keep it. there are only some symlinks in it to retrieve git commit etc.

Reply:
SOUNDS GOOD TO ME (famous last words :rofl: )